katas/currying_multiply_elements.js

```diff
@@ -0,0 +1,6 @@
+//From https://www.codewars.com/kata/586909e4c66d18dd1800009b
+
+let multiplyAll =
```

Comment:
Function parameters should be on same line, use `const` instead of `let` when the value should not change.